changelog/7685.improvement.md

```diff
@@ -0,0 +1,2 @@
+* Added warnings for competing entity extractors at training time and for overlapping entities at inference time
```

Comment:
Mhm, I think this could breeak the generation of the changelog file. I think this would at least need a title before starting the bullet pointed list.